Examples/Scripts/Python/full_chain_itk.py

```diff
@@ -97,15 +96,13 @@
     s,
     trackingGeometry,
     field,
-    CKFPerformanceConfig(ptMin=1.0 * u.GeV if ttbar_pu200 else 0.0, nMeasurementsMin=6),
-    TrackSelectorRanges(pt=(1.0 * u.GeV, None), absEta=(None, 4.0)),
+    TrackSelectorConfig(pt=(1.0 * u.GeV, None), absEta=(None, 4.0), minMeasurements=6),
```

Comment:
true but that will only change the output of the CKF performance writer and I also also wanted to align the minimum pT cutoffs everywhere

Reply:
Ok, but it becomes completely untraceable what changes the output? We also don't automatically run the ttbar atm, so we can't tell how this change that?

Reply:
the downstream algorithms should be unaffected by the `CKFPerformanceConfig`. what will change is the amount of tracks the vertexing will get because of the `minMeasurements=6`.

what will also change is the IO output of the CKF because tracksummary and trackstates will also be effected by the filter. not sure how bad this is for people who use the full chain but this was the idea to make the pulls unbiased by outlier tracks